core/tee/tee_rpmb_fs.c

```diff
@@ -749,7 +754,7 @@ static TEE_Result tee_rpmb_resp_unpack_verify(struct rpmb_data_frame *datafrm,
 	memcpy(&lastfrm, &datafrm[nbr_frms - 1], RPMB_DATA_FRAME_SIZE);
 
 	/* Handle operation result and translate to TEEC error code. */
-	bytes_to_u16(lastfrm.op_result, &op_result);
+	get_op_result_bits(lastfrm.op_result, &op_result);
```

Comment:
`op_result` should be `uint8_t`. Also it seems `rawdata::op_result` is essentially useless so better remove it to make this more clear.

Reply:
Done